shared/src/main/scala/Interpreter.scala

```diff
@@ -5,6 +5,10 @@ import vyxal.impls.Elements
 import VNum.given
 
 object Interpreter:
+  def runLiterate(code: String)(using ctx: Context): Unit =
```

Comment:
Do we want to add a `literate: Boolean = false` parameter to `Interpreter.execute` itself instead of making a separate method?

Reply:
I guess it depends on what would be more concise to call in other files. I'm fine either way I think - I just made it a separate method because I wanted to avoid fricking up everything else.